scalableminds/webknossos
# Compute proper bucket priorities for flight mode

### URL of deployed dev instance (used for testing):
- https://flightmodepriorities.webknossos.xyz

### Steps to test:
- open flight mode for dataset
- throttle internet speed in devtools
- ensure that buckets are somewhat loading inside-out (don't expect it to be super precise as in ortho mode, since the fallback rendering can intervene here)

### Issues:
- fixes #3931 

------
